packages/jni-swig-stub/src/main/jni/realm_api_helpers.cpp

```diff
@@ -0,0 +1,333 @@
+/*
```

Comment:
Like discussed, should we move these files outside the `jni-swig-stub` folder or find a pattern so we don't accidentally delete files here?